airsonic-main/src/main/java/org/airsonic/player/security/GlobalSecurityConfig.java

```diff
@@ -162,7 +162,7 @@ protected void configure(HttpSecurity http) throws Exception {
                     .and().authorizeRequests()
                     .antMatchers("/recover*", "/accessDenied*",
                             "/style/**", "/icons/**", "/flash/**", "/script/**",
-                            "/sonos/**", "/crossdomain.xml", "/login", "/error")
```

Comment:
This is unrelated to this PR, correct?

Reply:
Nope, it's related: since we're not using Flash anymore, we can remove the crossdomain.xml file.